### PR TITLE
Fix issue with forking and few typo fixes

### PR DESCRIPTION
**Pull request by slide**
- Do not allow forking a repository that is named as an existing repository in the org.
- Fix typo in kick message
- Fix incorrect message when trying to add a user to a team

### CODE REVIEW

File: src/main/java/org/jenkinsci/backend/ircbot/IrcBotImpl.java [hunk 1]
```diff
@@ -403,6 +403,9 @@ private void setupHosting(String channel, String sender, String hostingId, Strin
                     + "* [Accept the invitation to the Jenkins CI Org on Github|https://github.com/jenkinsci]\n"
                     + "* [Request upload permission|https://wiki.jenkins-ci.org/display/JENKINS/Hosting+Plugins#HostingPlugins-Requestuploadpermissions]\n"
                     + "* [Releasing your plugin|https://wiki.jenkins-ci.org/display/JENKINS/Hosting+Plugins#HostingPlugins-Releasingtojenkins-ci.org]\n"
+                    + "\n\nIn order for your plugin to be built by the Jenkins CI infrastructure and check PR's, please add a [Jenkinsfile|https://jenkins.io/doc/book/pipeline/jenkinsfile/] to the root "
```
Comment (daniel-beck) [inline src/main/java/org/jenkinsci/backend/ircbot/IrcBotImpl.java]: Should link to the ci.jenkins.io index page

File: src/main/java/org/jenkinsci/backend/ircbot/IrcBotImpl.java [hunk 2]
```diff
@@ -825,6 +828,12 @@ boolean forkGitHub(String channel, String sender, String owner, String repo, Str
                 return false;
             }
 
+            check = org.getRepository(repo);
+            if(check != null) {
+                sendMessage(channel, "Repository "+repo+" can't be forked, an existing repository with that name already exists in "+IrcBotConfig.GITHUB_ORGANIZATION);
+                return false;
```
Comment (daniel-beck) [inline src/main/java/org/jenkinsci/backend/ircbot/IrcBotImpl.java]: Thank you!

File: src/main/java/org/jenkinsci/backend/ircbot/IrcBotImpl.java (hunk 1 above)
Comment (slide) [reply src/main/java/org/jenkinsci/backend/ircbot/IrcBotImpl.java]: You mean the "Jenkins CI Infrastructure" part of the string?

Comment (daniel-beck) [reply src/main/java/org/jenkinsci/backend/ircbot/IrcBotImpl.java]: yes

**Pull request merged**